index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
- [ ] It would be better if an empty space is not left in your code. Leaving empty spaces does not make your code to follow HTML && CSS best practices